packages/dd-trace/src/plugins/ci_plugin.js

```diff
@@ -77,6 +80,24 @@ module.exports = class CiPlugin extends Plugin {
         }
       })
     })
+
+    this.addSub(`ci:${this.constructor.id}:itr:skipped-suites`, ({ skippedSuites, frameworkVersion }) => {
+      const testCommand = this.testSessionSpan.context()._tags[TEST_COMMAND]
+      skippedSuites.forEach((testSuite) => {
+        const testSuiteMetadata = getTestSuiteCommonTags(testCommand, frameworkVersion, testSuite, this.constructor.id)
+
+        this.tracer.startSpan(`${this.constructor.id}.test_suite`, {
+          childOf: this.testModuleSpan,
+          tags: {
+            [COMPONENT]: this.constructor.id,
+            ...this.testEnvironmentMetadata,
+            ...testSuiteMetadata,
+            [TEST_STATUS]: 'skip',
+            [TEST_SKIPPED_BY_ITR]: 'true'
```

Comment:
An off-topic question: does CI test cycle protocol support boolean literals? We could save some traffic here and probably in other places

Reply:
that's a good question. I don't think so but I'll confirm.